### ISSUE TITLE
Add `estimatefee`  to RPC commands

### ISSUE THREAD

**Issue by RdeWilde** [cevap/ion#70]
Make `estimefee` available thru RPC, like bitcoin does:
https://bitcoin.org/en/developer-reference#estimatefee

In my particular case it would be usefull for electrum-server

**Comment by cevap** [comment]
Assigned to [#card-4014795](../projects/16#card-4014795)

**Comment by mitchellcash** [comment]
Why is there a wontfix label?

**Comment by aspaas** [comment]
I removed the label, but I'm not promising a deadline at this moment!

**Comment by mitchellcash** [comment]
👍  submitting an issue is not committing the maintainers to a deadline, so I think the removal makes sense

**Comment by mitchellcash** [comment]
Original PR in Bitcoin that adds estimatefee / estimatepriority RPC methods:

https://github.com/bitcoin/bitcoin/pull/3959

**Comment by aspaas** [comment]
True, and besides, nothing is stopping anybody from submitting a PR themselves before I get to it ;)
Thanks for the suggestion, good one!

**Comment by cevap** [comment]
wontfix: from perspective of issue creator, I only assumed it is not fixable until we provide it, then it will become fixable

**Comment by cevap** [comment]
deadline is set by accorging milestone
